README.md

```
@@ -1,36 +1,183 @@
-yarn dev
-# or
-pnpm dev
-# or
-bun dev
+## 💬 프로젝트 소개
+
+내 사람과의 약속을 쉽게 관리할 수 있는 서비스 **사람, 별**
+<br><br>
+
+> - **작업 기간** : 2025.03.28 ~ 2025.05.01
+> - **배포 주소** : https://www.sarambyeol.com
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)